### DISCUSSION TITLE
The "Quickdraw" achievement

### DISCUSSION THREAD

**Discussion by uwussimo** [Schweinepriester/github-profile-achievements#136] [Q&A] [answered]
Any hints to get "Quickdraw"? I tried to create an issue with my friend and then let me close it nicely, but it didn't work. Also, I tried to close my friends (even my own) PR. However, it didn't work out either.

**Comment by thenithinbalaji** [comment, marked as answer]
It should work. Did you close it within 5 mins? Try creating an issue in your own repo and close it instantly.
Example,
https://github.com/thenithinbalaji/Text-Encrypter/issues/1

**Reply by uwussimo** [reply]
can i do the same thing on your repo?

**Reply by thenithinbalaji** [reply]
yay, you got the badge @uwussimo

**Reply by uwussimo** [reply]
ladies and gentlemen, we got'em! 🎉

**Comment by Nirzak** [comment]
I still remember I created a pull request and closed it withing 5 minutes and also my issue got closed within 5 minutes before but still didn't get the badge!

**Comment by WixkedLuck** [comment]
1. Create a repository
2. Create an "issue" and fill out forms 
3.  Click back on issue tab
4. Click on the issue you create and click the button solve issue and fill out forms
